games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,117 @@
+SUMMARY="An Amazing 3D Flight Simulator"
+DESCRIPTION="FlightGear creates a sophisticated flight simulator framework \
+for use in research or academic environments, pilot training, as an industry \
+engineering tool, for DIY-ers to pursue their favorite interesting flight \
+simulation idea, and last but certainly not least as a fun, realistic, and \
+challenging desktop flight simulator. We are developing a sophisticated, open \
+simulation framework that can be expanded and improved upon by anyone interested \
+in contributing."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/flightgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="61f809ef0a3f6908d156f0c483ed5313d31b5a6ac74761955d0b266751718147"
+PATCHES="flightgear-2018.2.2.patchset"
+ADDITIONAL_FILES="flightgear.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	flightgear$secondaryArchSuffix
+	cmd:fgcom
+	cmd:fgelev
+	cmd:fgfs
+	cmd:fgjs
+	cmd:fgpanel
+	cmd:fgtraffic
+	cmd:fgviewer
+	cmd:GPSsmooth
+	cmd:JSBSim
+	cmd:js_demo
+	cmd:metar
+	cmd:MIDGsmooth
+	cmd:UGsmooth
+	cmd:yasim
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libglew$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
```

Comment:
```diff
 	lib:libcurl$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
 	lib:libfreetype$secondaryArchSuffix
 	lib:libgl$secondaryArchSuffix
 	lib:libglew$secondaryArchSuffix
 	lib:libglu$secondaryArchSuffix
-	lib:libpng$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
```
(I'm not sure about libexecinfo, but it might be needed.)